### PR TITLE
Gradebook styling issues

### PR DESCRIPTION
**Pull request by matthugs**
JIRA: https://openedx.atlassian.net/browse/EDUCATOR-4622
along with an issue I haven't stuck in our bug spreadsheet (described below)

before
Screen Shot 2019-09-09 at 1 57 53 PM
after
Screen Shot 2019-09-10 at 2 02 28 PM
Screen Shot 2019-09-10 at 2 02 04 PM

### CODE REVIEW

**Comment by matthugs** [comment]
before for the ticketed bug:
Screen Shot 2019-09-10 at 2 16 05 PM
after:
Screen Shot 2019-09-10 at 2 16 29 PM

File: src/components/Gradebook/index.jsx [hunk 1]
```diff
@@ -831,16 +842,14 @@ export default class Gradebook extends React.Component {
                 onChange={this.handleMaxAssigGradeChange}
               />
               <span className="input-percent-label">%</span>
-              <div className="d-flex align-items-center">
```
Comment (iloveagent57) [inline src/components/Gradebook/index.jsx]: What does getting rid of this `<div>` do?

Comment (iloveagent57) [reply src/components/Gradebook/index.jsx]: Ah, you just added a screenshot that answers this question :)

Comment (matthugs) [reply src/components/Gradebook/index.jsx]: The div was an attempt to get the button into a better-aligned position, but it also complicated efforts to fix this bug. I figure it's also good to make the assignment grade filter and the course grade filter markup consistent with one another, also.

**Pull request merged**